_includes/search.html

```diff
@@ -0,0 +1,25 @@
+<script language="Javascript" type="text/javascript">
+  function my_search_google()
+  {
+    var query = document.getElementById("my-google-search").value;
+    window.open("http://google.com/search?q=" + query
+	+ "%20site:" + "http://theforeman.org");
+  }
+</script>
+
+<div class="container container-search">
+  <a class="pull-left" href="/feed.xml">
+    <img src="/static/images/rss.png" alt="RSS Feed" />
+    Get the RSS News Feed
```

Comment:
I like it more, certainly. There's some wasted whitespace above the searchbox now, but I think I'm ok with that. I guess the final option would be to put the RSS link and the search together on the right (one over the other) opposite the heading.

Added your commit to the PR as it's definitely an improvement. Thanks!